exapp/service.go

```diff
@@ -19,29 +21,34 @@ type Service interface {
 
 	RetrieveByID(string) (model.Event, error)
 
+	GetLogger() *logger.Logger
 }
 
 var _ Service = (*eventsService)(nil)
 
 type eventsService struct {
-	events EventsRepository
+	events events.EventsRepository
+	logger *logger.Logger
 	//	hasher Hasher
 	//	idp    IdentityProvider
 }
 
 // New instantiates the events service implementation.
-func New(events EventsRepository) Service {
-	return &eventsService{events: events}
+func New(events events.EventsRepository, logger *logger.Logger) Service {
+	return &eventsService{events, logger}
+}
+
+// GetLogger - retrieves logger that can be used
+func (svc eventsService) GetLogger() *logger.Logger {
```

Comment:
Go does not use ideioms like `Get` for function names (like Java does). Getters in go are called simply `Logger()`.

https://golang.org/doc/effective_go.html#Getters

```There's nothing wrong with providing getters and setters yourself, and it's often appropriate to do so, but it's neither idiomatic nor necessary to put Get into the getter's name. If you have a field called owner (lower case, unexported), the getter method should be called Owner (upper case, exported), not GetOwner.```

Also -is this function exported? If it is used only in this package, you should considder naming it with lowecase first letter - `logger()`.